pageserver/src/http/routes.rs

```diff
@@ -108,7 +108,7 @@ fn apierror_from_prerror(err: PageReconstructError) -> ApiError {
 fn apierror_from_tenant_map_insert_error(e: TenantMapInsertError) -> ApiError {
     match e {
         TenantMapInsertError::StillInitializing | TenantMapInsertError::ShuttingDown => {
-            ApiError::InternalServerError(anyhow::Error::new(e))
+            ApiError::InternalServerError(e.into())
```

Comment:
Let's either use `T::from` (if possible) to be more explicit what error we're converting into, or add a `.context` to the original error.

Curios, what was the purpose of this particular change? That does not seem to change things really?

Reply:
reverted